test/contributors/annotations/json_key/json_key_annotation_test.dart

```diff
@@ -1,8 +1,8 @@
 import 'dart:io' as io;
```

Comment:
Json key annotation tests failed for me.
They expect in-place generation, but got a generated file instead.